lib/ioki/model/operator/vehicle.rb

```diff
@@ -29,6 +29,11 @@ class Vehicle < Base
                   on:   :read,
                   type: :string
 
+        attribute :default_resource_configuration,
```

Comment:
why default_resource_configuration? Is there another resource configuration somewhere?

Reply:
That's what it is called in the VehicleSerializer. My understanding is, that a vehicle has one default resource configuration. To get all other resource configurations for a vehicle one has to use the new endpoints for resource configuration 👍